package.json

```diff
@@ -223,7 +223,6 @@
     "electron-updater": "2.20.1",
     "keytar": "4.1.0",
     "node-forge": "0.7.1",
-    "os-locale": "2.1.0",
```

Comment:
There's another package.json in src that needs this removed from as well.

Reply:
Done.